target/linux/mvebu/patches-4.14/403-net-mvneta-convert-to-phylink.patch

```diff
@@ -928,12 +928,13 @@ Signed-off-by: Russell King <rmk+kernel@arm.linux.org.uk>
  	return 0;
  
  err_netdev:
-@@ -4384,16 +4490,14 @@ err_netdev:
+@@ -4384,16 +4490,15 @@ err_netdev:
  		mvneta_bm_pool_destroy(pp->bm_priv, pp->pool_short,
  				       1 << pp->id);
  	}
 +	if (pp->phylink)
 +		phylink_destroy(pp->phylink);
++err_free_stats:
```

Comment:
change it by
`goto err_free_stats;` to `goto err_netdev;`
should be more proper

see https://github.com/x-wrt/x-wrt/commit/c7fca6ea64838d11c084ff736e7dc3e06e352129

Reply:
err_free_stats label was drop by upstream kernel
we may use label `err_netdev` instead